new-lamassu-admin/src/pages/Transactions/helper.js

```diff
@@ -21,11 +18,7 @@ const getCashInStatus = it => {
 }
 
 const getCashInStatusDetails = it => {
```

Comment:
The getCashInStatusDetails and getCashOutStatusDetails functions have the same body. Can turn these into a single function

Reply:
Although the bodies are the same, I believe it is important to keep them separated for readability's sake and as they serve different directions of transactions, their logic should be kept separated, despite being equal, for now.

Tell me what you think about this and if my argument has changed your mind about this

Reply:
I think that we can keep a single one while the logic is one and separate when they become different things in fact, if this ever occurs (may as well be never). By doing this we avoid, for example, updating one function while forgeting to change the other, which can cause some hard to track bugs. I agree that some readability may be lost in the process (even though while having only one behaviour keeping a single function is simpler and represents the current truth), but we avoid adding unnecessary complexity to the code, which can add up fast.